doc/source/user/quickstart.rst

```diff
@@ -628,10 +628,18 @@ An array has a shape given by the number of elements along each axis::
 
 The shape of an array can be changed with various commands::
 
-    >>> a.ravel() # flatten the array
+    >>> a.ravel() # returns the array, flattened
     array([ 2.,  8.,  0.,  6.,  4.,  5.,  1.,  1.,  8.,  9.,  3.,  6.])
-    >>> a.shape = (6, 2)
-    >>> a.T
+    >>> a.ravel().shape
+    (12,)
+    >>> a.reshape(6,2) # returns the array with a modified shape
+    array([[ 2.,  8.],
+           [ 0.,  6.],
+           [ 4.,  5.],
+           [ 1.,  1.],
+           [ 8.,  9.],
+           [ 3.,  6.]])
+    >>> a.T # returns the array after being transposed
```

Comment:
unfortunatly `a` does not have the `(6, 2)` since if you remove the .shape assignment, `a.reshape(6, 2)` will return a reshaped array, so if you assign it to a new `a = a.reshape(...)` it would work to replace `a.shape = ...`.

Not sure how its best, but I appreciate that you are taking the trouble of fixing a little bit more then initially.